src/api/index.js

```diff
@@ -18,3 +18,13 @@ export const getUserInfo = () => logined.get('user/my');
 
 export const signUp = ({ account, password, find_email, nickname }) =>
   logined.post('/user/sing-in', { account, password, find_email, nickname });
+
+
+export const keywordAPI = {
+  InquiryKeyword : () => logined.get(`/keyword`),

```

Comment:
`camelCase` 로 컨벤션 맞춰주시길 바랍니다.

```suggestion
  inquiryKeyword : () => logined.get(`/keyword`),
```

또는 간단히 `getKeyword` 해도 되겠네요
```suggestion
  getKeyword : () => logined.get(`/keyword`),
```

Reply:
넵 camelCase 지키도록 하겠습니다.
키워드 조회라는 이름에서 조회를 생각해서 inquiry를 사용했는데, 그냥 get을 사용하는 것이  더 좋은것 같습니다!

Reply:
3c5adff

Reply:
아하 그런 경우라면 API 요청부는 요청만을 의미하는 네이밍을 만들고, 
이를 사용하는 함수에서 `inquiryKeyword` 등으로 네이밍 해주면 좋을것 같습니다
API 요청 함수는 `요청` 이라는 문맥만 강조하는게 좋다고 생각합니다~